lib/src/circle_progress_bar/circle_progress_bar.dart

```diff
@@ -11,12 +11,14 @@ class CircleProgressBar extends StatefulWidget {
     this.reversedDirection = false,
     required this.foregroundColor,
     required this.value,
+    this.strokeWidth,
```

Comment:
How about writing `this.strokeWidth = 6,` here, then one can easily see the default stroke width. Please change and I will accept the pull request.

Thank you for making this library better.

Reply:
Yess, I updated move default value is 6 to contractor

Reply:
Thanks, merged and new version is published.